ReactWindows/ReactNative.Shared/Chakra/Executor/ChakraJavaScriptExecutor.cs

```diff
@@ -316,6 +329,31 @@ private JToken ConvertJson(JavaScriptValue value)
         }
         #endregion
 
+        #region Native Flush Queue Immediate Hook
+        private JavaScriptValue NativeFlushQueueImmediate(
+            JavaScriptValue callee,
+            bool isConstructCall,
+            JavaScriptValue[] arguments,
+            ushort argumentCount,
+            IntPtr callbackData)
+        {
+            if (argumentCount != 1)
+            {
+                throw new ArgumentOutOfRangeException(nameof(argumentCount), "Expected excactly one argument");
+            }
+
+            if (_flushQueueImmediate == null)
+            {
+                throw new InvalidOperationException("FlushQueueImmediate hook has not been set");
+            }
+
+            _flushQueueImmediate(ConvertJson(arguments[0]));
```

Comment:
>_flushQueueImmediate [](start = 12, length = 20)

Did you test this? Usually the first argument is the global argument, so I would expect at least two arguments.